### ISSUE TITLE
Server error on users bread

### ISSUE THREAD

**Issue by AndrewVSutherland** [LMFDB/lmfdb#4532]
https://beta.lmfdb.org/users/ throws a server error

```
Exception on /users/ [GET]
Traceback (most recent call last):
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/sage/sage-9.2/local/lib/python3.6/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/users/main.py", line 111, in list
    user_rows = tuple(zip(*[users[i*n: (i + 1)*n] for i in range(COLS)]))
  File "/home/lmfdb/lmfdb-git-dev/lmfdb/users/main.py", line 111, in <listcomp>
    user_rows = tuple(zip(*[users[i*n: (i + 1)*n] for i in range(COLS)]))
TypeError: slice indices must be integers or None or have an __index__ method
[2021-04-20 17:25:01 UTC] 500 error on URL https://beta.lmfdb.org/users/ ()
```

**Comment by JohnCremona** [comment]
Could be a python3 issue?

**Comment by AndrewVSutherland** [comment]
Closed via #4549
